content/en/docs/concepts/workloads/pods/_index.md

```diff
@@ -138,6 +138,23 @@ it is deleted.
 When you create the manifest for a Pod object, make sure the name specified is a valid
 [DNS subdomain name](/docs/concepts/overview/working-with-objects/names#dns-subdomain-names).
 
+### Pod OS
+
+{{< feature-state state="stable" for_k8s_version="v1.25" >}}
+
+You should set the `.spec.os.name` field to either `windows` or `linux` to indicate the OS on
```

Comment:
nit:
Should this be 'To indicate the OS on which you want to run the pod on set the `.spec.os.name` field to either `windows` or `linux`. ?
This avoids language like `should` or `can`.

Reply:
I like that we're making a recommendation. It's more than “there's this field, you could use it if you like”; now the field is GA, it's “you ought to set this field”.

_If we had the capacity_ it'd be nice to show a sample webhook that produces a [Warning](https://kubernetes.io/blog/2020/09/03/warnings/) when you don't set that for a Pod / pod template.

Reply:
Given that there's `System.Net.HttpListener`, that sample webhook could even be written in PowerShell!

Reply:
> If we had the capacity it'd be nice to show a sample webhook that produces a [Warning](https://kubernetes.io/blog/2020/09/03/warnings/) when you don't set that for a Pod / pod template.

We can do that but it is left to the user as an exercise since the use-cases may differ.